### ISSUE TITLE
Give an ability to select available icons in Tokens Edit screen

### ISSUE THREAD

**Issue by irina060981** [alpheios-project/alignment-editor-new#613]
after online discussion with @abrasax

- we could define an options - what icons would be visible for the user - in Tokens Edit Options
- we could give an ability to select by one, or by groups

@monzug , I created this issue to remember this task - but I believe we would need to have some final discussions before starting to work on it

**Comment by irina060981** [comment]
From email with @abrasax 
decided to change to groups

In other words something like

active icons:

[ ]  add and delete newlines
[ ]  add and delete tokens
[ ]  merge or split tokens
[]   edit tokens

**Comment by irina060981** [comment]
Form the email with requirements in pdf
it is done on Text Enter screen

**Comment by irina060981** [comment]
![image](https://user-images.githubusercontent.com/12377640/152497350-f2c73ce8-b239-4b07-be41-4660f04b9bf4.png)

By default 
   Enable Tokens Editor Screen and all icons groups are false
Advanced
   Enable Tokens Editor Screen and all icons groups are true

**Comment by irina060981** [comment]
Done

**Comment by monzug** [comment]
Looks good. if annotation is on but all the edit options are not checked, I cannot do anything with the tokens in edit text. I am ok with this. We could also required to have minimum one option checked but I do not think it's necessary.

**Comment by irina060981** [comment]
I agree, and I thought about it - but decided to not add additional complexity without a discussion.

**Comment by monzug** [comment]
I see your point. let's bring it up at the next discussion

**Comment by irina060981** [comment]
Created a separate issue
